src/editor.c

```diff
@@ -324,6 +339,18 @@ static gboolean on_editor_button_press_event(GtkWidget *widget, GdkEventButton *
 				keybindings_send_command(GEANY_KEY_GROUP_GOTO, GEANY_KEYS_GOTO_MATCHINGBRACE);
 			return TRUE;
 		}
+		if (event->type == GDK_BUTTON_PRESS && event->state == GDK_MOD1_MASK)
```

Comment:
don't use `event->state` directly (at least not when comparing to equality), because it can contain a whole lot of problematic things (numlock state, etc.).  Luckily, there's `state` already available here :)

Reply:
> don't use event->state directly (at least not when comparing to equality), because it can contain a whole lot of problematic things (numlock state, etc.). Luckily, there's state already available here :)

Yes, but `keybindings_get_modifiers()` filters-out Alt (`GDK_MOD1_MASK`) I wanted to use.

Reply:
Does it? It filters <kbd>MOD2</kbd>, but I don't see any kind that it does this for MOD1?  Actually quite the contrary, it's in the default modmask for all platforms, and even again for macos (looking at GTK3's `gtk_accelerator_get_default_mod_mask()` implementation -- and the respective platform vfuncs).

Reply:
Alright, it's possible it was filtering-out the strange modifier I was getting on my linux VM running under macOS where I can't properly develop this (which will be the case in the upcoming week when I'm out of home with my macOS machine only).